### ISSUE TITLE
Linter not Using Python3

### ISSUE THREAD

**Issue by utkonos** [SublimeLinter/SublimeLinter-pydocstyle#12]
I have pep8, pyflakes, and pydocstyle all installed, but for some reason pydocstyle tries to use python2 rather than python3. I am unable to change this. Here is the debug output from console:

```
SublimeLinter: find_python(version=None, script='pydocstyle', module=None) 
SublimeLinter: find_python: default python = /usr/local/bin/python 
SublimeLinter: find_python: available version = {'major': 2, 'minor': 7} 
SublimeLinter: find_python: 'pydocstyle' path = /usr/local/bin/pydocstyle 
SublimeLinter: find_python: <= ('/usr/local/bin/python', '/usr/local/bin/pydocstyle', 2, 7) 
SublimeLinter: pydocstyle version query: /usr/local/bin/python /usr/local/bin/pydocstyle --version 
SublimeLinter: WARNING: no pydocstyle version could be extracted from:
Traceback (most recent call last):
  File "/usr/local/bin/pydocstyle", line 7, in <module>
    from pydocstyle.cli import main
ImportError: No module named pydocstyle.cli
```

Both of the other SublimeLinter plugins use a different find_python, and they get python3 correctly:
`find_python(version='3', script=None, module=<module 'pyflakes.api' from '/usr/local/lib/python3.6/site-packages/pyflakes/api.py'>)`
and
`find_python(version='3', script=None, module=<module 'pep8' from '/usr/local/lib/python3.6/site-packages/pep8.py'>)`

This problem appears to be the same as this one:
https://github.com/SublimeLinter/SublimeLinter-pylint/issues/12

**Comment by brbsix** [comment]
It seems the linter needs to be implemented as module rather than a script in order for it to function properly.

Here's a really basic example: https://github.com/brbsix/SublimeLinter-pydocstyle/commit/f16f5527599d8b54a7cc01e418e975a8ef182bd2

Here's something more usable: https://github.com/brbsix/SublimeLinter-pydocstyle/commit/97c481fe8a81edb6a575536debd5ad17752add3a

In the latter example, I monkey patch pydocstyle in order for it to accept command-line arguments. In this case only `--add-ignore` is supported though the other options can be added very easily.

**Comment by utkonos** [comment]
@brbsix Can you create a good pull request so that the project owners can merge it?

**Comment by brbsix** [comment]
@utkonos I just submitted one here: https://github.com/SublimeLinter/SublimeLinter-pydocstyle/pull/14

Unfortunately I don't have the time to look it over more closely, but hopefully it's enough to get the job done.

**Comment by groteworld** [comment]
Closed with the addition of #14
